### PR TITLE
Fix bug to show the secondary avatar when the repost is is a policy expense chat

### PR DESCRIPTION
**Pull request by wlegolas**
### Details

When I user requests money and splits it into a workspace, in workspace chat the split bill avatar displays the default profile sub avatar and on hovering on it, it displays the same user tooltip.

### Fixed Issues

$ https://github.com/Expensify/App/issues/29614
PROPOSAL: https://github.com/Expensify/App/issues/29614#issuecomment-1763214403

### Tests

1. Open the app and login with user A
2. Open the app on another device and login with user B
3. From user A, invite user B to any workspace
4. From user B, click on green plus and click on request money
5. Enter any amount and continue
7. Press the "Split" button beside the workspace name, press "Add to split" and confirm the split summary
8. From user A, open workspace chat was created after inviting user B to a workspace

- [x] From user A, in workspace chat, observe the secondary (small) avatar, ensure that the initial letter is from the workspace name, and when hovering, ensure that the tooltip shows the workspace name.

### Offline tests

1. Open the app and login with user A
2. Open the app on another device and login with user B
3. From user A, invite user B to any workspace
4. From user B, click on green plus and click on request money
5. Enter any amount and continue
6. Press the "Split" button beside the workspace name, press "Add to split" and confirm the split summary
7. From user A, open workspace chat was created after inviting user B to a workspace
9. Switch to offline mode

- [x] From user A, in workspace chat, observe the secondary (small) avatar, ensure that the initial letter is from the workspace name, and when hovering, ensure that the tooltip shows the workspace name.

### QA Steps

1. Open the app and login with user A
2. Open the app on another device and login with user B
3. From user A, invite user B to any workspace
4. From user B, click on green plus and click on request money
5. Enter any amount and continue
6. Press the "Split" button beside the workspace name, press "Add to split" and confirm the split summary
7. From user A, open workspace chat was created after inviting user B to a workspace

- [x] From user A, in workspace chat, observe the secondary (small) avatar, ensure that the initial letter is from the workspace name, and when hovering, ensure that the tooltip shows the workspace name. 

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android: Native
    - [x] Android: mWeb Chrome
    - [x] iOS: Native
    - [x] iOS: mWeb Safari
    - [x] MacOS: Chrome / Safari
    - [x] MacOS: Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] If we are not using the full Onyx data that we loaded, I've added the proper selector in order to ensure the component only re-renders when the data it is using changes
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>

https://github.com/Expensify/App/assets/698363/371688ab-7e19-4fbb-93f7-acacc680b317

</details>

<details>
<summary>Android: mWeb Chrome</summary>

https://github.com/Expensify/App/assets/698363/4c6c6d6d-94a3-4c48-ba11-61cbe2c9041b

</details>

<details>
<summary>iOS: Native</summary>

https://github.com/Expensify/App/assets/698363/aab33de4-f4a3-4c10-bb09-9f3a0a71cc04

</details>

<details>
<summary>iOS: mWeb Safari</summary>

https://github.com/Expensify/App/assets/698363/d8ecc1a4-ea1e-46bf-b7ab-4e11433a87c4

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

https://github.com/Expensify/App/assets/698363/33d3bd3c-6bb0-4af7-8079-94c68075984a

Added tests to validate this issue
image

</details>

<details>
<summary>MacOS: Desktop</summary>

https://github.com/Expensify/App/assets/698363/b3e1f154-036a-4416-959d-78cb25507539

</details>

### CODE REVIEW

**Comment by cubuspl42** [comment]
@wlegolas Ping me when you finish the checklist

**Comment by wlegolas** [comment]
> @wlegolas Ping me when you finish the checklist

Sure @cubuspl42, I'm trying to run the mobile app but I've been having problems running the IOS and Android apps.

Do you know how I can have access to Stackoverflow for example this URL here https://stackoverflow.com/c/expensify/questions/13283/13284#13284?

**Comment by cubuspl42** [comment]
@wlegolas Typically, external contributors don't access Expensify's StackOverflow Teams. If you're having technical issues, you can start a thread on Slack. You can mention me in that thread.

**Comment by wlegolas** [comment]
> @wlegolas Typically, external contributors don't access Expensify's StackOverflow Teams. If you're having technical issues, you can start a thread on Slack. You can mention me in that thread.

Hi @cubuspl42 I requested access to Slack, when I get the access I'll start a thread.

Thank you

Just another question, about doing a test with a “**high traffic**” account, I requested to change my account and did the same steps to validate my fix, and worked.

Is there a specific test scenario that I have to do with the high traffic?

**Comment by wlegolas** [comment]
> @wlegolas Don't bother too much with the high-traffic tests here. There are some issues where this is important, but it nearly certainly won't affect us here.
> 
> The last checkbox of the checklist is quite tricky...
> 
> > I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.
> 
> Sometimes, the checkboxes don't apply to a given PR. It's always better to be biased toward the interpretation that they _do_, but if it's clear that they don't, well, then they don't.
> 
> Let's focus on testing all the supported platforms (with videos).

Hi @cubuspl42 thank you for clarifying and for all context.

I've been having problems running the app on Android, in the README there is the step below, but this link is related to Expensify's StackOverflow Teams. 

> * Go through the instructions on [this SO post](https://stackoverflow.com/c/expensify/questions/13283/13284#13284) to start running the app on android.

I requested access to Slack but I haven't received my invite yet. I'm waiting for this access to finish this last test.

If you have any information that helps me with this, I'll be happy to know.

**Comment by cubuspl42** [comment]
@wlegolas 

Have you tried running `npm run android`?

After you do, go for a walk or pick up some other work, as it takes an eternity (especially if you don't own high-end hardware).

**Comment by wlegolas** [comment]
@cubuspl42.

I finished the tests on all devices and checked, and I finished my checklist, thank you for your help on it. 

One question, when I tried to run the application for IOS, I needed to configure my environment, I followed the section "**Running the iOS app** 📱" in the README file, but to solve the problem with file permission I used this article [How to solve FilePermissionError after installing rbenv on Mac](https://dev.to/heymarkkop/how-to-solve-filepermissionerror-after-installing-rbenv-on-mac-13ij).

Could I update the README file by adding this article? If you disagree, don't worry, I'm just sharing this because it helped me a lot to solve my problem.

**Comment by cubuspl42** [comment]
About updating README, you can share your experience on Slack when you have access and see what others think.

**Comment by wlegolas** [comment]
> About updating README, you can share your experience on Slack when you have access and see what others think.

@cubuspl42 

Makes more sense.

I'm done, if you have any questions or suggestions, please let me know.

Thank you for your support.

**Comment by cubuspl42** [comment]
You should make clear what's the _expected_ behavior in the testing steps. You can check how testing steps look in other PRs, [this is my PR](https://github.com/Expensify/App/pull/25524#issue-1857697526) I had at hand.

**Comment by wlegolas** [comment]
Thank you for your feedback @cubuspl42.

I thought that the last step was the expected behavior that we wanted, so I changed it to have the expected behavior as a checkbox item.

**Comment by cubuspl42** [comment]
## Reviewer Checklist

- [x] I have verified the author checklist is complete (all boxes are checked off).
- [x] I verified the correct issue is linked in the `### Fixed Issues` section above
- [ ] I verified testing steps are clear and they cover the changes made in this PR
    - [x] I verified the steps for local testing are in the `Tests` section
    - [x] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [x] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I verified tests pass on **all platforms** & I tested again on:
    - [x] Android / native
    - [x] Android / Chrome
    - [x] iOS / native
    - [x] iOS / Safari
    - [x] MacOS / Chrome / Safari
    - [x] MacOS / Desktop
- [x] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [x] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>

https://github.com/Expensify/App/assets/2590174/f64fba62-f132-4d80-a6c5-2294c861438a

</details>

<details>
<summary>Mobile Web - Chrome</summary>

https://github.com/Expensify/App/assets/2590174/f5088ccd-f3a3-4358-bf6f-787f68bf1e62

</details>

<details>
<summary>Mobile Web - Safari</summary>

https://github.com/Expensify/App/assets/2590174/a3a0b550-71a6-4572-9bb0-3532f43a981d

</details>

<details>
<summary>Desktop</summary>



</details>

<details>
<summary>iOS</summary>

https://github.com/Expensify/App/assets/2590174/5fb8a86c-67f6-46f4-8ccc-8cbde024d70f

</details>

<details>
<summary>Android</summary>

https://github.com/Expensify/App/assets/2590174/6010665a-ed4c-4a96-bb77-29edb2c16442

</details>

**Comment by cubuspl42** [comment]
>  From user A, in workspace chat, observe the split bill avatar, it displays the initial letter from the workspace name, and when hovering, it displays the tooltip with the workspace name

We have a primary (large) and secondary (small) avatar, we should make clear which is which. I think that "split bill avatar" is a ambiguous

**Comment by cubuspl42** [comment]
> Click on the split button beside workspace report used in step 3 and complete further process

-> Press the "Split" button beside the workspace name, press "Add to split" and confirm the split summary

**Comment by wlegolas** [comment]
> > From user A, in workspace chat, observe the split bill avatar, it displays the initial letter from the workspace name, and when hovering, it displays the tooltip with the workspace name
> 
> We have a primary (large) and secondary (small) avatar, we should make clear which is which. I think that "split bill avatar" is a ambiguous

@cubuspl42 

I changed it to be more clear using your suggestion, would you validate if is more clear to you, please?

**Comment by cubuspl42** [comment]
It's better

>  it displays 

You're saying what "does" happen, while we should phrase it as "_Ensure_ that..."

**Review by jakub-trzebiatowski — APPROVED** [review]

File: tests/unit/ReportActionItemSingleTest.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    PERSONAL_DETAILS_LIST: 'personalDetailsList',
+    IS_LOADING_REPORT_DATA: 'isLoadingReportData',
+    COLLECTION: {
+        REPORT_ACTIONS: 'reportActions_',
+        POLICY: 'policy_',
+    },
+    NETWORK: 'network',
+};
+
+describe('ReportActionItemSingle', () => {
```
Comment (cubuspl42) [inline tests/unit/ReportActionItemSingleTest.js:17]: @techievivek I approved this, but I'd like to double-check if you think the added tests are well-formed and fit well into Expensify conventions. This is the first big unit test I've received as a C+.

**Comment by wlegolas** [comment]
@cubuspl42 thank you so much for all your suggestions.

**Comment by cubuspl42** [comment]
> @cubuspl42 thank you so much for all your suggestions.

This is my job 🙃

**Comment by cubuspl42** [comment]
@techievivek Do you have an option to manually re-restart that check?

**Comment by wlegolas** [comment]
Hi @cubuspl42, do we also need to manually re-restart this step "**PR Reviewer Checklist / checklist (pull_request_review)**"?

**Comment by cubuspl42** [comment]
## Reviewer Checklist

- [x] I have verified the author checklist is complete (all boxes are checked off).
- [x] I verified the correct issue is linked in the `### Fixed Issues` section above
- [x] I verified testing steps are clear and they cover the changes made in this PR
    - [x] I verified the steps for local testing are in the `Tests` section
    - [x] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [x] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I verified tests pass on **all platforms** & I tested again on:
    - [x] Android / native
    - [x] Android / Chrome
    - [x] iOS / native
    - [x] iOS / Safari
    - [x] MacOS / Chrome / Safari
    - [x] MacOS / Desktop
- [x] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [x] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>

https://github.com/Expensify/App/assets/2590174/f64fba62-f132-4d80-a6c5-2294c861438a

</details>

<details>
<summary>Mobile Web - Chrome</summary>

https://github.com/Expensify/App/assets/2590174/f5088ccd-f3a3-4358-bf6f-787f68bf1e62

</details>

<details>
<summary>Mobile Web - Safari</summary>

https://github.com/Expensify/App/assets/2590174/a3a0b550-71a6-4572-9bb0-3532f43a981d

</details>

<details>
<summary>Desktop</summary>



</details>

<details>
<summary>iOS</summary>

https://github.com/Expensify/App/assets/2590174/5fb8a86c-67f6-46f4-8ccc-8cbde024d70f

</details>

<details>
<summary>Android</summary>

https://github.com/Expensify/App/assets/2590174/6010665a-ed4c-4a96-bb77-29edb2c16442

</details>

**Comment by cubuspl42** [comment]
@wlegolas Oh my, so many checks are failing. Please ensure the "Lint code" and "Verify signed commits" aren't right here.

The "CLA Assistant" and "PR Reviewer Checklist" seem to be out-of-date, unless I'm missing something

File: tests/unit/ReportActionItemSingleTest.js (hunk 1 above)
Comment (techievivek) [reply tests/unit/ReportActionItemSingleTest.js:17]: Yeah, it looks good to me —great job @wlegolas, for adding the tests.

File: tests/utils/LHNTestUtils.js [hunk 2]
```diff
@@ -167,6 +170,63 @@ function getAdvancedFakeReport(isArchived, isUserCreatedPolicyRoom, hasAddWorksp
...
+ */
+function getFakeReportWithPolicy(participantAccountIDs = [1, 2], millisecondsInThePast = 0, isUnread = false) {
+    return {
+        ...getFakeReport(participantAccountIDs, millisecondsInThePast, isUnread),
+        type: CONST.REPORT.TYPE.CHAT,
+        chatType: CONST.REPORT.CHAT_TYPE.POLICY_EXPENSE_CHAT,
+        policyID: '08CE60F05A5D86E1',
+        oldPolicyName: '',
+        isOwnPolicyExpenseChat: false,
+        ownerAccountID: 1,
```
Comment (techievivek) [inline tests/utils/LHNTestUtils.js:187]:
```suggestion
        ownerAccountID: participantAccountIDs[0],
```
should this depend on `participantAccountIDs` instead?

File: tests/utils/LHNTestUtils.js [hunk 3]
```diff
@@ -167,6 +170,63 @@ function getAdvancedFakeReport(isArchived, isUserCreatedPolicyRoom, hasAddWorksp
...
+    };
+}
+
+/**
+ * @param {Number} [id]
+ * @param {String} [name] the number of milliseconds in the past for the last message timestamp (to order reports by most recent messages)
```
Comment (techievivek) [inline tests/utils/LHNTestUtils.js:193]: The doc for the param doesn't look correct.

**Review by techievivek — COMMENTED** [review]
Looks good to me, there are a couple of things you will need to complete as a first time contributor.

**Comment by techievivek** [comment]
Let me re-run the travis as it looks like @wlegolas has also signed the CLA.

**Comment by techievivek** [comment]
@wlegolas There seems to be some prettier diff for your code. Can you run the command `npm run prettier` and commit the changes? Also, you will need to sign your commits https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#begin-coding-your-solution-in-a-pull-request

File: tests/unit/ReportActionItemSingleTest.js (hunk 1 above)
Comment (wlegolas) [reply tests/unit/ReportActionItemSingleTest.js:17]: Hi @techievivek thank you.

File: tests/utils/LHNTestUtils.js (hunk 3 above)
Comment (cubuspl42) [reply tests/utils/LHNTestUtils.js:193]: It definitely doesn't... I missed this.

File: tests/utils/LHNTestUtils.js (hunk 2 above)
Comment (wlegolas) [reply tests/utils/LHNTestUtils.js]: @techievivek you're right, I changed it in this commit [751a380c](https://github.com/Expensify/App/pull/30068/commits/751a380c3e85ee12c299e1ec5122d2553785031c)

File: tests/utils/LHNTestUtils.js (hunk 3 above)
Comment (wlegolas) [reply tests/utils/LHNTestUtils.js]: Sorry my bad, I copied and pasted the doc from another method.

Done [b9a9f8f](https://github.com/Expensify/App/pull/30068/commits/b9a9f8f19b20aa7f97ec0db0b85e24de395765b7)

**Comment by wlegolas** [comment]
> @wlegolas There seems to be some prettier diff for your code. Can you run the command `npm run prettier` and commit the changes? Also, you will need to sign your commits https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#begin-coding-your-solution-in-a-pull-request

@techievivek I fixed the Prettier issues, you can see the changes in the commit [df0bc8861](https://github.com/Expensify/App/pull/30068/commits/df0bc8861d593782999442ba016dd31dbdb670fd)

**Comment by wlegolas** [comment]
Hi @cubuspl42 and @techievivek I finished all suggestions and settings. Looks like now the only required step is the PR approval.

Is this PR ok for you to give me approval?

**Review by techievivek — APPROVED** [review]
Looks good, thanks for the changes and tests. 🙇

**Comment by techievivek** [comment]
Going to merge it since @cubuspl42 already approved it earlier.

**Pull request merged**